hal/halx86/generic/dma.c

```diff
@@ -1551,6 +1553,7 @@ HalAllocateAdapterChannel(IN PADAPTER_OBJECT AdapterObject,
 
             KeReleaseSpinLock(&MasterAdapter->SpinLock, OldIrql);
 
+            // TODO: Is it fine if ExAllocatePoolWithTag() failed?
```

Comment:
yeah strange. This would mean that the map cannot be grown "for later"... @binarymaster @disean @ThFabba @tkreuzer What do you think about that?